spec/integration/rspec/match_matcher_spec.rb

```diff
@@ -445,11 +445,11 @@
             diff: proc {
               plain_line %|  [|
               plain_line %|    "milk",|
-              plain_line %|    "toast",|
```

Comment:
These are tests around `collection_including`. That's not the same thing as `contains_exactly`. Therefore, it's not these tests that should be updated, but those for `collection_including`.

Reply:
Yes, I looked again and realized that I was misunderstanding it from before. I think everything's good then.